src/components/atoms/Toast/Toast.tsx

```diff
@@ -0,0 +1,40 @@
+import { Text, useToast, Flex, IconButton } from '@chakra-ui/react'
+import { ReactElement, useEffect, useState } from 'react'
+import { CloseIcon } from '../CloseIcon/CloseIcon'
+import { ToastVariants } from '@/@types/Toast'
+
+interface ToastProps {
+  message: string
+  icon: ReactElement
```

Comment:
Can it be of "Icon" type from chakra?

Reply:
ReactElement, lets us pass directly our own icons if use chakraicon it will not allow us to use the one from design.

Reply:
Ah yeah, it's not an inheritance.